pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -477,22 +478,18 @@ func (r *ReconcileBareMetalHost) actionInspecting(prov provisioner.Provisioner,
 		return result, nil
 	}
 
+	if details != nil {
+		info.host.Status.HardwareDetails = details
+		result.Requeue = true
```

Comment:
Should this set RequeueAfter as well?

Reply:
I don't see any reason to delay - we're not waiting for anything external at this point. We just want to write the new data and then requeue immediately so that we can check if there are any other steps that need to be taken.

Reply:
Yeah, that makes sense.